conf/gulp-tasks/library.gulpfile.js

```diff
@@ -17,7 +18,7 @@ const { generateProcessTranslationJsCall } = require('../i18n/runtimecallgenerat
  * @returns {Promise<Function>}
  */
 exports.dev = function devJSBundle () {
-  return createBundleTaskFactory(DEV_LOCALE).then(devTaskFactory => {
+  return createBundleTaskFactory(DEFAULT_LOCALES[0]).then(devTaskFactory => {
```

Comment:
I think we should make it so that `DEFAULT_LOCALES` isn't an array. Right now, we just have plans for `build` to run against one locale. It's a bit onerous to have to add `[0]` whenever you want to get the default locale. If we decide later on that we want `build` to generate assets for a couple of locales, we can revisit making this an array. For now, I feel it adds unnecessary complexity for this to be an array.

Reply:
Sure. I can make it singular.